src/Symfony/Component/Cache/PhpMarshaller/Reference.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Cache\PhpMarshaller;
+
+/**
+ * @author Nicolas Grekas <p@tchwork.com>
+ *
+ * @internal
+ */
+class Reference
+{
+    public function __construct(int $id)
+    {
+        $this->{0} = $id;
```

Comment:
why using a dynamic property rather than a class property ?

Reply:
when var_export() will work on these, it will generate packed arrays, which consume less memory